nautobot/core/views/mixins.py

```diff
@@ -369,6 +378,8 @@ class BulkCreateForm(BootstrapMixin, Form):
                 csv_file = CSVFileField(from_form=self.bulk_create_form_class)
 
             form_class = BulkCreateForm
+        elif self.action == "notes":
+            form_class = NoteForm
         else:
             form_class = getattr(self, f"{self.action}_form_class", None)
```

Comment:
Nit, but maybe we could default `notes_form_class = NoteForm` on the class and avoid the need to add an `elif self.action == "notes"`? Would allow for the (admittedly unlikely) case where a view wants to provide a custom notes form too.

Reply:
On the `NautobotUIViewSet` class?